src/structures/session.ts

```diff
@@ -708,22 +738,29 @@ export default abstract class Session {
         this.connection.stopPlaying();
 
         try {
-            let inputArgs = ["-ss", seekLocation.toString()];
+            let inputArgs = ["-ss", seekLocation!.toString()];
             let encoderArgs: Array<string> = [];
             const specialType = this.isListeningSession()
                 ? null
                 : this.guildPreference.gameOptions.specialType;
 
             if (specialType) {
                 const ffmpegArgs = specialFfmpegArgs[specialType](
-                    seekLocation,
+                    seekLocation!,
                     songDuration,
                 );
 
                 inputArgs = ffmpegArgs.inputArgs;
                 encoderArgs = ffmpegArgs.encoderArgs;
             }
 
+            if (clipAction) {
+                const durationLimit =
+                    this.guildPreference.gameOptions.guessTimeout;
```

Comment:
Guess timeout is being used as the amount of time the song plays? This doesn't seem right

Reply:
I think this decision has led to weak points in this PR. However, I still think it makes the most sense for the end user. They are already familiar with changing the duration of a song with `/timer`.

If we move this value in as an argument to `/play`, the user loses the ability to customize this value during the game, and this will lead to shorter games with lost unique song history.

If we introduce another command scoped just for this, users may attempt to change its value using `/timer` and would need to learn another command. It would also mean we would need to track its state in the database, ignore showing it in `/options` except for during a clip game, and as a result, would not have an elegant way of changing its value outside a game.